polar-wasm-api/src/query.rs

```diff
@@ -73,9 +73,8 @@ impl Query {
     }
 
     #[wasm_bindgen(js_class = Query, js_name = source)]
-    pub fn wasm_source(&self) -> JsResult<JsValue> {
-        let source = self.0.source_info();
-        serde_wasm_bindgen::to_value(&source).map_err(|e| serialization_error(e.to_string()))
+    pub fn wasm_source(&self) -> String {
```

Comment:
This cleanup is unrelated to the rest of the PR; just noticed that the type of `Query.source()` was `any` instead of `string` and wondered why.